packages/web-app-user-management/tests/unit/components/Users/DeleteUserModal.spec.ts

```diff
@@ -48,16 +42,17 @@ describe('DeleteUserModal', () => {
 })
 
 function getWrapper({ propsData = {} } = {}) {
-  return mount(DeleteUserModal, {
-    localVue,
-    propsData: {
-      ...propsData
-    },
-    stubs: { 'oc-modal': true, 'oc-text-input': true },
-    mocks: {
-      $gettextInterpolate: (translation) => translation,
-      $ngettext: (translationSingular, translationPlural, count) =>
-        count > 1 ? translationPlural : translationSingular
-    }
-  })
+  return {
+    wrapper: shallowMount(DeleteUserModal, {
+      props: { ...propsData },
+      global: {
+        mocks: {
+          $gettextInterpolate: (translation) => translation,
+          $ngettext: (translationSingular, translationPlural, count) =>
```

Comment:
Wouldn't it make sense to move this mock implementation to `defaultComponentMocks` and then use them here?